heron/tools/ui/resources/static/js/plan-stats.js

```diff
@@ -60,6 +60,16 @@
         scaleTicks: [0, 500, 1000, 1500, 2000],
         format: function (d) { return d.toFixed(0); }
       },
+      {
+        name: 'BP',
```

Comment:
@objmagic ok I will try to adjust the CSS and try again with 'Back Pressure' label.